src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Add comments for your interface. Especially when your interface contains no code, comments are really necessary for other's to know what this interface would do.